chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -668,4 +680,19 @@ private void checkUploadedSurveillanceOwnership(Surveillance pendingSurv) {
 			} 
 		}
 	}
+	
+	private void updateAuthority(Surveillance surv){
```

Comment:
Why is it ONC_STAFF here? I thought it was ONC_ADMIN and ACB_ADMIN or ACB_STAFF that we cared about.